src/promptflow/promptflow/_sdk/entities/_trace.py

```diff
@@ -258,9 +259,14 @@ def _from_spans(spans: typing.List[Span]) -> typing.Optional["LineRun"]:
         if main_line_run_data is None:
             return None
 
-        evaluations = dict()
+        evaluations = []
         for eval_name, eval_line_run_data in evaluation_line_run_data_dict.items():
-            evaluations[eval_name] = eval_line_run_data
+            evaluations.append(
+                {
+                    EvaluationKeyName.NAME: eval_name,
```

Comment:
why not just: evaluations.append(eval_line_run_data)
eval_line_run_data should have the eval_name already?

Reply:
yes, just synced with Kaiyi, we should avoid that name